template/partial/mythbusters/myth_3.html

```diff
@@ -0,0 +1,3 @@
+ASSUMPTION: The culture here is different, so Peace Corps staff will blame me for what happened.
+<div class="ui hr divider"></div>
```

Comment:
Shouldn't the div enclose this description?
Also try not to put plain text directly in html files. It's better to enclose them in some tag